contracts/v2/PolygonRollupManager.sol

```diff
@@ -237,7 +237,7 @@ contract PolygonRollupManager is
         keccak256("EMERGENCY_COUNCIL_ADMIN");
 
     // Current rollup manager version
-    string public constant ROLLUP_MANAGER_VERSION = "al-v0.3.1";
+    string public constant ROLLUP_MANAGER_VERSION = "v2.1.0";
```

Comment:
Why not starting with version v1.0.0?

Reply:
As I say in the comment on this PR:
> For contracts that previously had the version al-v0.3.0 and already existed, I’ve chosen v2.0.0 (also because of the v2 folder they are located in).

I think it makes more sense, but open to suggestions!^

Reply:
I would use v1.0.0, I guess in a future clean of the contracts folders the v2 wil be removed.

Reply:
I think `v1` fits better here.
At the end, we are now doing just a mapping from `al-v0.3.1` to `vX.Y.Z`.
Since it is the first version with the new versioning, `v1` may fit better here.
The folder name should be agnostic to the versioning.